closure/bin/build/jscompiler.py

```diff
@@ -97,12 +97,17 @@ def _GetJsCompilerArgs(compiler_jar_path, java_version, source_paths,
   # Add the application JAR.
   args += ['-jar', compiler_jar_path]
 
-  for path in source_paths:
-    args += ['--js', path]
+  #collect list of compiler flags
+  compiler_flags.extend(['--js ' + path for path in source_paths])
+  if jvm_flags:
+    compiler_flags.extend(jvm_flags)
+
+  #write flags to file for avoiding problem with too big arguments length
+  flags_file = tempfile.NamedTemporaryFile(delete = False)
```

Comment:
`delete = False` means this file won't be deleted even after the program exits, correct? I'd prefer not to pollute the user's disk as someone could be compiling very often.

Can you write this in such a way that it cleans up after itself? For example, rm'ing the file after compilation.